.github/workflows/test-packages.yml

```diff
@@ -23,6 +23,7 @@ jobs:
     - name: Set up QEMU
       uses: docker/setup-qemu-action@v3
       with:
+        image: tonistiigi/binfmt:qemu-v7.0.0-28
```

Comment:
I _think_ you actually need the "full" image tag here, starting with `docker.io/`: see how the default is handled [here](https://github.com/docker/setup-qemu-action/blob/master/src/context.ts#L12).

Reply:
Hmm, interesting. I think that's technically more correct, however I've seen other repositories fix it like this without adding the `docker.io/` prefix.